### DISCUSSION TITLE
The downsampling ratio of VAE in maisi

### DISCUSSION THREAD

**Discussion by DopamineLcy** [Project-MONAI/tutorials#1841] [Q&A]
The downsampling ratio of VAE in maisi is 4, i.e. a [128, 128, 128] patch results in a [32, 32, 32] latent.
While 32 ^ 3 = 32768 is very large for a transformer-based model.
Is there any pre-trained VAE with a larger down-sampling ratio like 8 ([128, 128, 128]->[16,16,16]?

Thank you!

**Comment by Can-Zhao** [comment]
We currently do not have 8x8x8 downsampling VAE. Thank you for letting us know that this is desired!
